### PR TITLE
Write EW on stack_plot in plots.py

### PR DESCRIPTION
**Pull request by marijana777**

### CODE REVIEW

File: linetools/analysis/plots.py [hunk 1]
```diff
@@ -95,7 +93,15 @@ def stack_plot(abslines, vlim=[-300,300.]*u.km/u.s, nrow=6, show=True, spec=None
             ax.get_xaxis().set_ticks([])
         # Label
         ax.text(0.1, 0.1, iline.data['name'], transform=ax.transAxes, ha='left', va='center')#, fontsize='large')  # , bbox={'facecolor':'white'})
-
+        if add_ew:
+            ewval = '{:05.2f}'.format(iline.attrib['EW'].value)
+            ewsigval = '{:05.2f}'.format(iline.attrib['sig_EW'].value)
+            compsign = ' = '
+            if iline.analy['flg_eye'] == 1:
```
Comment (profxj) [inline linetools/analysis/plots.py]: Let's also show as an upper limit if EW < 3*sig_EW

File: linetools/analysis/plots.py [hunk 2]
```diff
@@ -95,7 +93,15 @@ def stack_plot(abslines, vlim=[-300,300.]*u.km/u.s, nrow=6, show=True, spec=None
             ax.get_xaxis().set_ticks([])
         # Label
         ax.text(0.1, 0.1, iline.data['name'], transform=ax.transAxes, ha='left', va='center')#, fontsize='large')  # , bbox={'facecolor':'white'})
-
+        if add_ew:
+            ewval = '{:05.2f}'.format(iline.attrib['EW'].value)
+            ewsigval = '{:05.2f}'.format(iline.attrib['sig_EW'].value)
+            compsign = ' = '
```
Comment (profxj) [inline linetools/analysis/plots.py]: maybe remove the spaces here to pack the text in

File: linetools/analysis/plots.py (hunk 1 above)
Comment (profxj) [inline linetools/analysis/plots.py]: And in this case, have the value reported be 2*sig_EW

File: linetools/analysis/plots.py (hunk 2 above)
Comment (marijana777) [inline linetools/analysis/plots.py]: done

File: linetools/analysis/plots.py (hunk 1 above)
Comment (marijana777) [inline linetools/analysis/plots.py]: done

**Comment by profxj** [comment]
Looks good.  Am approving and will merge.

**Pull request merged**